projects/project.cpp

```diff
@@ -208,11 +213,11 @@ namespace projects {
       
       for (size_t p=0; p<getObjectWrapper().particleSpecies.size(); ++p) {
          this->setVelocitySpace(p,cell);
+         uint n = cell->get_number_of_velocity_blocks(p);
+         if ( n < 64) {
+            cout << " WARNING!!! Low block count in this cell for population " << getObjectWrapper().particleSpecies[p].name << ". Number of velocity blocks = " << n << endl;
```

Comment:
Should be printed to cerr I think. Other such warnings also go to cerr (e.g. the cubic cell warning).